saleor/webhook/payloads.py

```diff
@@ -396,6 +396,23 @@ def generate_product_variant_media_payload(product_variant):
     ]
 
 
+def generate_product_variant_with_stock_payload(stocks: Iterable["Stock"]):
```

Comment:
We should be sure that `Product` and `ProductVariant` are prefetched for this function.